sthgetter/sthgetter.go

```diff
@@ -85,7 +85,11 @@ func getCheckStoreSTH(ctx context.Context, lc *client.LogClient, sv *ct.Signatur
 		glog.Infof("%s: %s: %s", l.URL, logStr, b.String())
 	}
 
-	// TODO(katjoyce): Store STH & errs.
+	// Store STH & associated errors.
+	glog.Infof("%s: %s: writing STH...", l.URL, logStr)
```

Comment:
nit: We'll probably want to make some of this verbose logging.

Reply:
Yup fo' sho'.  It's-a coming - was basically waiting for there to be more stuff being logged before putting some of it at different verbosity levels.